src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/BlobAuditingTests.ps1

```diff
@@ -0,0 +1,834 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests setting and getting blob auditing policy with classic storage
+#>
+function Test-BlobAuditingUpdatePolicyWithClassicStorage
```

Comment:
The reason for doing this is so that only this test needs to create the classic storage client, this allows you to remove `GetStorageClient();` from the test setup for the remaining tests at https://github.com/Azure/azure-powershell/pull/4259/files#diff-2bb7aee19d8ece9557881e7a68c23ea6R29 which means that you can record just those tests without setting up RDFE credentials. (This is what I did recently when upgrading server api-version)